lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -60,6 +87,12 @@ module.exports = {
     };
 
     return {
+      ImportDeclaration(node) {
+        if (!emberImportAliasName) {
+          emberImportAliasName = ember.getEmberImportAliasName(node);
```

Comment:
The idea that there is only one import is flawed, we should account for something like:

```js
import Ember from 'ember';
import Em from 'ember';

export default Ember.Object.extend({
  foo: Em.A(),
})
```

Can you create a "bad" test for this?

---

Additionally, storing the value in module scope means that this is not isolated per-file. Specifically, running two different files one with `import Ember from 'ember'` and the other with `import Em from 'ember'` would not work.

Reply:
Sure thing! I can create a bad test for this.
Didn't really think that it would be shared across files. 

Thanks for you feedback! I'll make the changes as soon as I can

Reply:
Hey @rwjblue  I see that it's often assumed (in other rules) that there's only one ember import. What should we do about that. Im not entirely sure how to do a "bad/invalid" test for that matter because I think that i should write a valid one to check this case.